static/templates/recipient_row.hbs

```diff
@@ -7,14 +7,17 @@
               style="background: {{background_color}}; border-left-color: {{background_color}};"
               href="{{stream_url}}"
               title="{{#tr}}Narrow to stream &quot;{display_recipient}&quot;{{/tr}}">
-                {{! invite only lock }}
-                {{#if invite_only}}
+                {{~! Icons for invite-only/web-public streams ~}}
+                {{~#if invite_only ~}}
                     <i class="fa fa-lock recipient-row-stream-icon" title="{{t 'This is a private stream' }}" aria-label="{{t 'This is a private stream' }}"></i>
                 {{/if}}
-                {{#if is_web_public}}
+
+                {{~#if is_web_public ~}}
                     <i class="fa fa-globe recipient-row-stream-icon" title="{{t 'This is a web public stream' }}" aria-label="{{t 'This is a web public stream' }}"></i>
                 {{/if}}
-                {{display_recipient}}
+
+                {{~! Recipient (e.g. stream/topic or topic) ~}}
+                {{~{display_recipient}~}}
```

Comment:
Handlebars recognizes `{{~{` as the combination of the `{{~` syntax for [whitespace control](https://handlebarsjs.com/guide/expressions.html#whitespace-control) with the `{{{` syntax for [skipping HTML escaping](https://handlebarsjs.com/guide/#html-escaping). The latter is a security vulnerability that we just published as [CVE-2021-3853](https://blog.zulip.com/2022/01/19/cve-2021-3853/) and fixed in 3eb2791c3e9695f7d37ffe84e0c2184fae665cb6.

@showell FYI.